### PR TITLE
Update to 1.19.2

### PR DESCRIPTION
**Pull request by woalk**
Rudimentary tests show that this seems to work. I have not tested all mixins extensively.

[Alpha release JAR](https://github.com/woalk/gate-of-babylon/releases/tag/v1.7.0-alpha-1.19.2)

### CODE REVIEW

File: src/main/java/draylar/gateofbabylon/mixin/EnchantmentHelperMixin.java [hunk 1]
```diff
@@ -26,10 +26,10 @@ public class EnchantmentHelperMixin {
             at = @At(value = "INVOKE", target = "Ljava/util/List;isEmpty()Z", ordinal = 0),
             locals = LocalCapture.CAPTURE_FAILHARD
     )
-    private static void generateEnchantments(Random random, ItemStack stack, int level, boolean treasureAllowed, CallbackInfoReturnable<List> cir, List list, Item item, int i, float f, List<EnchantmentLevelEntry> list2) {
+    private static void generateEnchantments(Random random, ItemStack stack, int level, boolean treasureAllowed, CallbackInfoReturnable<List<EnchantmentLevelEntry>> cir, List<EnchantmentLevelEntry> list, Item item, int i, float f, List<EnchantmentLevelEntry> list2) {
```
Comment (MJRamon) [inline src/main/java/draylar/gateofbabylon/mixin/EnchantmentHelperMixin.java:29]: This and line 32 break enchantment table in Minecraft

**Review by MJRamon — CHANGES_REQUESTED** [review]
Broken enchantments

**Comment by Coosanta17** [comment]
Pls fix  broken enchantments

**Comment by MJRamon** [comment]
> Pls fix broken enchantments

The fix is to revert changes at lines 29 and 32

**Comment by Coosanta17** [comment]
> The fix is to revert changes at lines 29 and 32

Did it and it works thanks very much

[Here](https://github.com/Coosanta17/gate-of-babylon/releases/tag/v1.8.0-Alpha-1.19.2) is the fixed jar

**Comment by NovenSandrin** [comment]
Has anyone tried contacting the original author about taking over the mod for the purpose of officially porting it to new Minecraft versions?

I've been working on a new mod to add Mythic Metals versions of all of the Gate of Babylon items (I need to bring it to 1.19.2), so it would be sad to see this mod die.  The new fixed 1.19.2 jar should last a while, though.

Once I have my mod working in 1.19.2 I'll put it up on Github, unless someone wants to assist me with updating the version, in which case I'll upload it as is (I had it working alongside All of Fabric 4 for MC 1.17.1).  I see that there is an issue related to adding a Mythic Metals extension.

@Draylar Any comment? :)

**Comment by MJRamon** [comment]
> Has anyone tried contacting the original author about taking over the mod for the purpose of officially porting it to new Minecraft versions?
> 
> I've been working on a new mod to add Mythic Metals versions of all of the Gate of Babylon items (I need to bring it to 1.19.2), so it would be sad to see this mod die. The new fixed 1.19.2 jar should last a while, though.
> 
> Once I have my mod working in 1.19.2 I'll put it up on Github, unless someone wants to assist me with updating the version, in which case I'll upload it as is (I had it working alongside All of Fabric 4 for MC 1.17.1). I see that there is an issue related to adding a Mythic Metals extension.
> 
> @Draylar Any comment? :)

It's [MIT License](https://github.com/Draylar/gate-of-babylon/blob/1.18.2/LICENSE), so you don't need to ask a permission

**Comment by Draylar** [comment]
> Has anyone tried contacting the original author about taking over the mod for the purpose of officially porting it to new Minecraft versions?
> 
> I've been working on a new mod to add Mythic Metals versions of all of the Gate of Babylon items (I need to bring it to 1.19.2), so it would be sad to see this mod die. The new fixed 1.19.2 jar should last a while, though.
> 
> Once I have my mod working in 1.19.2 I'll put it up on Github, unless someone wants to assist me with updating the version, in which case I'll upload it as is (I had it working alongside All of Fabric 4 for MC 1.17.1). I see that there is an issue related to adding a Mythic Metals extension.
> 
> @Draylar Any comment? :)

Done and uploaded to CurseForge - thanks for the help, woalk. I'm hoping to do a project rebrand on the CurseForge page to make it look more polished (the pink CF icon has served its time😆), so if you end up publishing a support/compatibility library for GOB, feel free to let me know and I can put a banner on the main CF page for it.

**Comment by MJRamon** [comment]
> > Has anyone tried contacting the original author about taking over the mod for the purpose of officially porting it to new Minecraft versions?
> > I've been working on a new mod to add Mythic Metals versions of all of the Gate of Babylon items (I need to bring it to 1.19.2), so it would be sad to see this mod die. The new fixed 1.19.2 jar should last a while, though.
> > Once I have my mod working in 1.19.2 I'll put it up on Github, unless someone wants to assist me with updating the version, in which case I'll upload it as is (I had it working alongside All of Fabric 4 for MC 1.17.1). I see that there is an issue related to adding a Mythic Metals extension.
> > @Draylar Any comment? :)
> 
> Done and uploaded to CurseForge - thanks for the help, woalk. I'm hoping to do a project rebrand on the CurseForge page to make it look more polished (the pink CF icon has served its time😆), so if you end up publishing a support/compatibility library for GOB, feel free to let me know and I can put a banner on the main CF page for it.

Enchanting is broken with that commit. We discussed it earlier.

**Comment by Draylar** [comment]
> Enchanting is broken with that commit. We discussed it earlier.

Already fixed with 930259433db5d84071ca6a1e4ea16b9b6481b608 :)

**Comment by MJRamon** [comment]
> > Enchanting is broken with that commit. We discussed it earlier.
> 
> Already fixed with [9302594](https://github.com/Draylar/gate-of-babylon/commit/930259433db5d84071ca6a1e4ea16b9b6481b608) :)

Tested with this fix, didn't work for me for some reason

**Comment by Draylar** [comment]
> Tested with this fix, didn't work for me for some reason

What problem are you running into? Before the patch commit, I could not enchant anything - the table would show results, but I could not click to take anything out of the table. The commit I referenced fixed the problem on my end (but perhaps it is acting differently in production?).

**Comment by MJRamon** [comment]
> > Tested with this fix, didn't work for me for some reason
> 
> What problem are you running into? Before the patch commit, I could not enchant anything - the table would show results, but I could not click to take anything out of the table. The commit I referenced fixed the problem on my end (but perhaps it is acting differently in production?).

I am sorry, there was a silly mistake on my end. Enchanting is working.

Also, I've made a title logo for you. Use it if it's useful or ignore it if it's not :) https://github.com/Draylar/gate-of-babylon/issues/95

**Pull request merged**